README.md

```diff
@@ -246,10 +246,13 @@ You can add a virtual host entry (`server` block) by placing a .conf file in the
     }
 
     # Dockerfile:
-    RUN rm /etc/nginx/sites-enabled/default
-    ADD webapp.conf /etc/nginx/sites-enabled/webapp.conf
-    RUN mkdir /home/app/webapp
-    RUN ...commands to place your web app in /home/app/webapp...
+    ADD webapp.conf /etc/nginx/sites-enabled/default
+
+    # Copy your web app files to /home/app/webapp
+    COPY ...... /home/app/webapp/
```

Comment:
Docker will automatically create the destination folder tree, so no need to `RUN mkdir /home/app/webapp`. One lesser layer.